packages/fast-component-explorer/README.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+This is the documentation site for `@microsoft/fast-components-react-msft`.
+
+It is built using the `@microsoft/fast-tooling-react` and `@microsoft/fast-components-react-msft` packages and shows the components in a playground as well as the documentation for each component.
```

Comment:
Sounds weird to me... maybe:

It is built using the `@microsoft/fast-tooling-react` and `@microsoft/fast-components-react-msft` packages to view, configure and document each component.